src/environments/environment .nonprod.ts

```diff
@@ -0,0 +1,31 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```

Comment:
There's a space in your file name. That might cause problems later.

Reply:
fixed